src/test/java/teammates/test/cases/logic/CoursesLogicTest.java

```diff
@@ -1097,41 +1088,41 @@ public void testCreateCourseAndInstructor() throws Exception {
         a.isInstructor = true;
         accountsDb.updateAccount(a);
         
-        c.id = "invalid id";
+        CourseAttributes invalidCourse = new CourseAttributes("invalid id", "Fresh course for tccai");
         
         try {
-            coursesLogic.createCourseAndInstructor(i.googleId, c.id, c.name);
+            coursesLogic.createCourseAndInstructor(i.googleId, invalidCourse.getId(), invalidCourse.getName());
             signalFailureToDetectException();
         } catch (InvalidParametersException e) {
             AssertHelper.assertContains("not acceptable to TEAMMATES as a Course ID", e.getMessage());
         }
-        verifyAbsentInDatastore(c);
+        verifyAbsentInDatastore(invalidCourse);
         verifyAbsentInDatastore(i);
         
         ______TS("fails: error during instructor creation due to duplicate instructor");
         
-        c.id = "fresh-course-tccai";
+        CourseAttributes courseWithDuplicateInstructor = new CourseAttributes("fresh-course-tccai", "Fresh course for tccai");
         instructorsDb.createEntity(i); //create a duplicate instructor
         
         try {
-            coursesLogic.createCourseAndInstructor(i.googleId, c.id, c.name);
+            coursesLogic.createCourseAndInstructor(i.googleId, courseWithDuplicateInstructor.getId(), courseWithDuplicateInstructor.getName());
             signalFailureToDetectException();
         } catch (AssertionError e) {
             AssertHelper.assertContains("Unexpected exception while trying to create instructor for a new course", e.getMessage());
         }
-        verifyAbsentInDatastore(c);
+        verifyAbsentInDatastore(courseWithDuplicateInstructor);
 
         ______TS("fails: error during instructor creation due to invalid parameters");
 
         i.email = "ins.for.iccai.gmail.tmt";
 
         try {
-            coursesLogic.createCourseAndInstructor(i.googleId, c.id, c.name);
+            coursesLogic.createCourseAndInstructor(i.googleId, courseWithDuplicateInstructor.getId(), courseWithDuplicateInstructor.getName());
```

Comment:
Hey @YongJieYongJie , some of the lines are still a little lengthy (> 110 characters) and could still be shortened by adding line breaks. If you notice that the lines are wrapped around in the git diff view, like the line above, it is a good sign that it should be shortened :).